### PR TITLE
Charter initial draft

### PR DESCRIPTION
**Pull request by gregwhitworth**
-----------------------

**NOTE:** This is not currently building for me, not entirely sure why but the `/sharp/` module keeps failing no matter what I try. That said I'd like folks to begin reviewing the charter content while I figure out why it won't build. I WILL NOT merge this PR until we ensure it won't break the build.

---------------------

This change adds a more in-depth charter of specific goals and process information for the group. @levithomason is working on a draft of the spec stages so I have not included those. Any feedback is welcomed.

For an easy to read version you can see it in rendered md here: https://github.com/WICG/open-ui/blob/gwhit-charter/research/src/pages/charter.research.mdx

### CODE REVIEW

File: research/src/pages/charter.research.mdx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+# Vision
+
+The web deserves interoperability between component frameworks and design systems.
+The UI community is in a similar place as the industrial revolution prior to standardization.
+Untold amounts of human effort are exhausted globally every day on rebuilding identical components for web apps.
+
+This should not be the case.
+The UI community should standardize namings and structures for common components.
+We should also standardize a way of theming these components.
+We should set a path for existing solutions to converge and for browser to natively provide these things in the future.
```
Comment (nicholasrice) [inline research/src/pages/charter.research.mdx:17]:
```suggestion
We should set a path for existing solutions to converge and for browsers to natively provide these things in the future.
```

File: research/src/pages/charter.research.mdx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+We shouldn't have to rebuild a dropdown, modal dialog, split button, or other components before we build our products.
+
+# Goals
+
+These are the goals Open UI believes will realize the above vision.
+
+- Document component names as they exist today
+- A common language for describing UIs and design systems
+- Eventual browser standards for web app components
+- Converging designer processes and developer workflows
```
Comment (levithomason) [inline research/src/pages/charter.research.mdx:31]: Suggest we remove bullet 4, and remove "Eventual" from bullet 3.

File: research/src/pages/charter.research.mdx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+# Scope of work
+
+- Utilize descriptivism of common patterns to identify and standardize on:
+  - Standardize component names and parts
+  - Standardize states
+  - Standardize on inputs that transition states (eg: keyboard or pointer interaction)
```
Comment (levithomason) [inline research/src/pages/charter.research.mdx:38]: Suggest we add something for capturing "Standardize design system terminology".  Example:

image

There should be an industry name for this.

File: research/src/pages/charter.research.mdx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+- Cover for each component the necessary accessibilty requirements (this is not in place of
+  WIA-ARIA or WICAG but linking where appropriate to alleviate implementor pain)
+- Upon discovery of gaps in the web platform, denote those and raise relevant issues to
+  standards bodies
+
+# Out of scope
+
+- Standardization of default appearance of the component
```
Comment (levithomason) [inline research/src/pages/charter.research.mdx:46]:
```suggestion
- Standardization of any appearance of any component
```

File: research/src/pages/charter.research.mdx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+The below process aims for efficiency and asynchronous changes but also allows for transparency
+and opportunities to ensure consesus from the community.
+
+- Research should be done across the industry for each aspect of the component
+- Based on investigation, curate decisions and propose an Open UI resolution
+- Open a PR regarding your proposal
+- Two members of the community group need to review and sign off on the change\
+- Merge of the review sign off will be done by the editors of Open UI. Current editors are:
+  - Greg Whitworth
+  - Levi Thomason
```
Comment (levithomason) [inline research/src/pages/charter.research.mdx:59]: Should we say "Current editors are listed in the CODEOWNERS file"?  This way it stays fresh.

File: research/src/pages/charter.research.mdx [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+- Add label of `needs merge` for review by the community group and editors
+
+# Process for consensus
```
Comment (levithomason) [inline research/src/pages/charter.research.mdx:62]: Let's callout that "consensus" does not mean stage 3.

**Pull request merged**